bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/impl/GemsStrategy.scala

```diff
@@ -27,7 +27,10 @@ import edu.gemini.spModel.core.{Angle, MagnitudeBand}
 import scalaz._
 import Scalaz._
 
-object GemsStrategy extends AgsStrategy {
+trait GemsStrategy extends AgsStrategy {
+  // By default use the remote backend but it can be overriden in tests
+  private [impl] implicit val backend:VoTableBackend = RemoteBackend
```

Comment:
`GemsStrategy` should not be called directly only via `AgsStrategy` and the val is private. Note that on the last commit the `val backend` is not implicit